### DISCUSSION TITLE
Particles position and velocity is single precision or double precision

### DISCUSSION THREAD

**Discussion by liangt1995** [glotzerlab/hoomd-blue#1968] [General]
Hi community:
I want to ask what is the format of particles position and velocity during running, single precision or double precision. I want to update one particle's position like "-80.0 + 0.00000001 * timestep" or keep its velocity as 0.000002 \sigma/\tau(using nve and zero_force). 

I am using the double precision version of hoomd-blue 2.9.7。I found the position can not update right or the velocity can not work right. I wonder if it is the precision problem. 

My hoomd version may be old, but if it can work in new version, please let me konw.

Thank you very much.

**Comment by joaander** [comment]
I will investigate if you provide a minimal script the reproduces the problem that I can execute with HOOMD-blue 5.0.0. There are no known problems with double precision values in the latest release as I confirmed with the complete transcript of an interactive session in https://github.com/glotzerlab/hoomd-blue/discussions/1947#discussioncomment-11358150.

**Reply by liangt1995** [reply]
Thank you for your reply, I will try the latest release and see if it can work.
